### ISSUE TITLE
Don't default export `from(process.env)` for environments that don't have `process`

### ISSUE THREAD

**Issue by smblee** [evanshortiss/env-var#155]
Trying to use this lib with vite which uses `import.meta.env` as the default location for env vars and I get the following error

```
process is not defined
    at node_modules/env-var/env-var.js (env-var.js:55:23)
```

and looks like it's happening because the module exports `from(process.env)` by default and its failing since `process` doesn't actually exist within the vite environment i am in (sveltekit).
https://github.com/evanshortiss/env-var/blob/3f5689ed13b307e4389b7dcf642bac55738e534c/env-var.js#L55

The code:

```
import { from } from 'env-var';

const env = from(import.meta.env);

export const API_BASE_URL = env.get('VITE_API_BASE_URL').required().asUrlString();
```

**Comment by linearcombination** [comment]
I am running into this as well as I am using vite with svelte (not sveltekit).

**Comment by evanshortiss** [comment]
This is available in the beta release of version 8. You can install and test it using `npm install env-var@beta`. Feel free to reopen this issue if necessary.

**Comment by kenberkeley** [comment]
@evanshortiss May I ask when you will release the beta version?

**Comment by evanshortiss** [comment]
@kenberkeley I have released v7.4.0 with this fix. You can now do the following for Vite environments:

```
const env = from(import.meta.env);
```

Or in React with `create-react-app`, you need to explicitly reference each variable like:

```
const env = from({
  SOME_VARIABLE: process.env. REACT_APP_SOME_VARIABLE
});
```
